extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java

```diff
@@ -1002,7 +1000,11 @@ public boolean isGroupCoordinator() {
 
     @Override
     public String getUDN() {
```

Comment:
Is there a specific reason for defining an udn member variable (Line 163), assigning this a value, but never use that again afterwards? why not calling getConfigAs() etc. each time, and be sure to have the latest value each time (even as it does not change ofter or at all)?